### PR TITLE
add use case section to ITE-9

### PR DESCRIPTION
**Pull request by pxp928**
Signed-off-by: pxp928 <parth.psu@gmail.com>

Based on the discussion in our last in-toto attestation maintainer meeting, new predicates needs to include a valid use case section to determine what it solves and how the current predicates (if any) don't solve this use case.  

cc @marcelamelara @TomHennen @adityasaky

### CODE REVIEW

File: ITE/9/README.adoc [hunk 1]
```diff
@@ -59,6 +59,10 @@ The new attestation definition SHOULD include the following fields.
 
 This section must discuss at a high level the reason for creating the new attestation type. Essentially, it describes the problems that can be solved using the new attestation.
 
+==== Use case
```
Comment (adityasaky) [inline ITE/9/README.adoc:62]:
```suggestion
==== Use Cases
```

**Review by adityasaky — APPROVED** [review]
LGTM!

File: ITE/9/README.adoc [hunk 2]
```diff
@@ -59,6 +59,10 @@ The new attestation definition SHOULD include the following fields.
...
+
+This section must discuss the use case that will be solved by this predicate and how the current predicate types (if any) do not solve this.
```
Comment (marcelamelara) [inline ITE/9/README.adoc]: I think "solved" is probably too strong a word here (but also adding a little more concreteness).

```suggestion
This section must discuss at least one concrete use case that will be covered by this predicate and how the current predicate types (if any) do not solve this.
```

**Review by marcelamelara — COMMENTED** [review]
LGTM, just a small wording change. Since ITE-9 is still in draft, this change doesn't need to be reviewed formally again, does it? 

cc/ @adityasaky

**Review by marcelamelara — APPROVED** [review]

File: ITE/9/README.adoc [hunk 3]
```diff
@@ -59,6 +59,10 @@ The new attestation definition SHOULD include the following fields.
 
 This section must discuss at a high level the reason for creating the new attestation type. Essentially, it describes the problems that can be solved using the new attestation.
 
+==== Use Cases
+
+This section must discuss at least one concrete use case that will be covered by this predicate and how the current predicate types (if any) do not solve this.
```
Comment (trishankatdatadog) [inline ITE/9/README.adoc:64]: Is this a note to self? If so, maybe mark as `TODO` or `FIXME`?

Comment (pxp928) [reply ITE/9/README.adoc:64]: No this is a new section in the ITE-9

Comment (adityasaky) [reply ITE/9/README.adoc:64]: It's a description of expected fields in a predicate proposal.

Comment (adityasaky) [reply ITE/9/README.adoc:64]: On that note, should we combine `Purpose` and `Use Cases` into one?

**Review by JustinCappos — APPROVED** [review]
LGTM

File: ITE/9/README.adoc (hunk 3 above)
Comment (adityasaky) [reply ITE/9/README.adoc:64]: > On that note, should we combine Purpose and Use Cases into one?

Actually, I think while they're related, it's fine to have them separate at slightly different levels of detail.

**Review by trishankatdatadog — APPROVED** [review]

**Pull request merged**